causalpy/pymc_models.py

```diff
@@ -1,3 +1,17 @@
+"""
+Defines generic PyMC ModelBuilder class and subclasses for
+
+- WeightedSumFitter model for Synthetic Control experiments
+- LinearRegression model
+
+Models are intended to be used from inside an experiment
+class (see `pymc_experiments.py
+<https://causalpy.readthedocs.io/en/latest/api_pymc_experiments.html>`_).
```

Comment:
If this link can be done using sphinx rather than a hard-coded URL, that would be even better. Not 100% sure, but this might help: https://stackoverflow.com/questions/20059953/linking-back-to-a-source-code-file-in-sphinx

Reply:
Thanks for the suggestion, switched to using the `:doc:` role which is much cleaner.